source/examples/xgboost-rf-gpu-cpu-benchmark/notebook.ipynb

```diff
@@ -0,0 +1,353 @@
+{
```

Comment:
<p>Can you format the commands using the Markdown code block? <a href="https://www.markdownguide.org/extended-syntax/#fenced-code-blocks" target="_blank">Use three backticks</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/252/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>